<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>691</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>PLACEHOLDER TEXT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="generalTab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QLineEdit" name="launchOptionsField">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>601</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Launch options:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="updatesTab">
       <attribute name="title">
        <string>Updates</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="localFilesTab">
       <attribute name="title">
        <string>Local Files</string>
       </attribute>
       <widget class="QPushButton" name="verifyIntegrityBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>191</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Verify Integrity</string>
        </property>
       </widget>
       <widget class="QPushButton" name="uninstallBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>191</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Uninstall</string>
        </property>
       </widget>
       <widget class="QPushButton" name="browseGameFilesBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>191</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse Game Files</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="langaugeTab">
       <attribute name="title">
        <string>Language</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="compatibilityTab">
       <attribute name="title">
        <string>Compatibility</string>
       </attribute>
       <widget class="QCheckBox" name="enableProtonBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>221</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable compatibility for this title</string>
        </property>
       </widget>
       <widget class="QComboBox" name="compatToolBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>221</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>531</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Changing the compat tool doesn't immediately unlock Windows launch options.
Usually this is because of a Proton update, please be patient as the update is downloaded.</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="betasTab">
       <attribute name="title">
        <string>Betas</string>
       </attribute>
       <widget class="QComboBox" name="betasDropdown">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>171</width>
          <height>25</height>
         </rect>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>71</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Select beta:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>231</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Access beta(s) with a beta key:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="betaKeyBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>171</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="testBetaKeyButton">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>130</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Check code</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
